acceptance-tests/src/test/resources/secp256k1/rich_benefactor_one.json

```diff
@@ -0,0 +1 @@
+{"address":"fe3b557e8fb62b89f4916b721be55ceb828dbd73","crypto":{"cipher":"aes-128-ctr","ciphertext":"c330870080626a6cf84c36eeb89f6376316637fbc16c86400eb6c394de5aed81","cipherparams":{"iv":"7f2526feb38ac355c644a84e78ec5692"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"b0da609bb541df918eb55bbeaf237445aa406a0c2d6e975663c4fcb221bc54e0"},"mac":"1ed0ef9e2092d0f5db8d16409ee2aaee9af3f521a5eafd910ce19dbc76e67ed7"},"id":"43f8e00a-c6ec-4a62-b7cb-f54cb299c3db","version":3}
```

Comment:
do we really need a "rich benefactor" for eth2signer? Or is this simply a validator?